components/feature/prompts/src/main/java/mozilla/components/feature/prompts/dialog/AutofillEditText.kt

```diff
@@ -0,0 +1,27 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+package mozilla.components.feature.prompts.dialog
+
+import android.content.Context
+import android.os.Build
+import android.util.AttributeSet
+import android.view.ViewStructure
+import androidx.appcompat.widget.AppCompatEditText
+
+internal class AutofillEditText : AppCompatEditText {
+    internal var sessionUrl: String = ""
+
+    constructor (context: Context) : super(context)
+
+    constructor (context: Context, attrs: AttributeSet?) : super(context, attrs)
+
+    constructor (context: Context, attrs: AttributeSet?, defStyleAttr: Int) : super(context, attrs, defStyleAttr)
+
+    override fun onProvideAutofillStructure(structure: ViewStructure?, flags: Int) {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
```

Comment:
Do you know how this supposed to work prior to 26?

Reply:
No, Autofill framework was only available starting API 26
https://developer.android.com/guide/topics/text/autofill